### ISSUE TITLE
I have a question about mmlu score

### ISSUE THREAD

**Issue by gyuwon12** [LZY-the-boys/HELM-Extended-Local#1]
I know MMLU has a lot of subsets, so are you simply going to add 57 EM values and average them?
![123](https://github.com/LZY-the-boys/HELM-Extended-Local/assets/99570746/97859d50-fe3d-4446-ae0e-b914d681f4ca)

**Comment by LZY-the-boys** [comment]
Yes. This table took a long time to get (57tasks 1000items per task)
